vision_agent_tools/models/model_registry.py

```diff
@@ -1,19 +1,63 @@
-from typing import Dict, Callable
+from typing import List, Dict, Type, Callable
+from enum import Enum
 from vision_agent_tools.shared_types import BaseMLModel
 from vision_agent_tools.models.florencev2 import Florencev2
+from vision_agent_tools.models.owlv2 import Owlv2
 
 
-MODEL_REGISTRY: Dict[str, Callable[[], BaseMLModel]] = {"florence2": Florencev2}
+class TextToObjectDetectionModel(str, Enum):
+    FLORENCE_V2 = "florencev2"
+    OWLV2 = "owlv2"
 
 
-def get_model(model_name: str) -> BaseMLModel:
+MODEL_REGISTRY: Dict[str, Callable[[], BaseMLModel]] = {
+    TextToObjectDetectionModel.FLORENCE_V2.value: Florencev2,
+    TextToObjectDetectionModel.OWLV2.value: Owlv2,
```

Comment:
@hugohonda if the dictionary is called `MODEL_REGISTRY` why are we using here a Tool based enum class to define the `MODEL_REGISTRY`?
It should be better to define an Enum with all the model names or put the model name directly as a string.

Reply:
you are right, fixed to put directly the string